extensions/amp-youtube/amp-youtube.md

```diff
@@ -60,11 +60,11 @@ E.g. in https://www.youtube.com/watch?v=Z1q71gFeRqM Z1q71gFeRqM is the video id.
 
 **data-param-***
 
-All `data-param-*` attributes will be added as query parameter to the youtube iframe src. This may be used to pass custom values through to youtube plugins, such as autoplay.
+All `data-param-*` attributes will be added as query parameter to the youtube iframe src. This may be used to pass custom values through to youtube plugins, such as whether to show controls.
 
 Keys and values will be URI encoded. Keys will be camel cased.
 
-- `data-param-autoplay=1` becomes `&autoplay=1`
```

Comment:
Can we also explicitly mention in the docs that the `autoplay` option is not allowed?